authentik/stages/email/stage.py

```diff
@@ -71,7 +72,7 @@ def get_token(self) -> FlowToken:
         valid_delta = timedelta(
             minutes=current_stage.token_expiry + 1
         )  # + 1 because django timesince always rounds down
-        identifier = slugify(f"ak-email-stage-{current_stage.name}-{pending_user}")
+        identifier = slugify(f"ak-email-stage-{current_stage.name}-{str(uuid4())}")
```

Comment:
@BeryJu what is the impact of this change on other places?

Reply:
nothing, existing tokens will continue to work, and this only applies to newly created tokens